lib/tmdb/client.rb

```diff
@@ -11,6 +11,8 @@ class Error < StandardError; end
 
     class << self
       def request(endpoint, params)
+        return { results: [] } if params.blank?
+        return { results: [] } if params[:query] == ''
```

Comment:
This is the client logic to resolve [TMDB-MOVIEQUEUE-6Z](https://mike-vallano.sentry.io/issues/3901921697/?project=127874&query=is%3Aunresolved&referrer=issue-stream) error when a user submits an empty string.